### PR TITLE
Adds missing JSDocs for build tasks

### PR DESCRIPTION
**Pull request by jaboc83**
JSDoc the last couple build task functions for consistency.

### CODE REVIEW

**Comment by erwinmombay** [comment]
LGTM!

File: build-system/tasks/lint.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ var options = {
   plugins: ['eslint-plugin-google-camelcase'],
 };
 
+/**
+ * Run the eslinter on the src javascript and log the output
+ * @return {!Object} Readable stream
```
Comment (erwinmombay) [inline build-system/tasks/lint.js]: actually might be more correct to switch this to `{!Stream}`

Comment (jaboc83) [inline build-system/tasks/lint.js]: That makes sense.

**Comment by erwinmombay** [comment]
@jaboc83 mind repushing, we have some flakey tests :(

**Comment by erwinmombay** [comment]
LGTM

**Pull request merged**